### ISSUE TITLE
Uninstall script

### ISSUE THREAD

**Issue by JohannesGaessler** [DistributedTaskScheduling/JobAdder#152]
Currently there is a script for installing JobAdder that utilizes pip.
However, uninstalling JobAdder via pip does not remove all components that were previously installed.
Another script for uninstalling JobAdder would be useful.

**Comment by ammen99** [comment]
Agreed, although it should be very easy - we install 4 binary files and the rest can be removed via pip (I don't think we should uninstall config files, since the user might have modified them and is just reinstalling job adder).
